Mathlib/Tactic/Check.lean

```diff
@@ -56,4 +105,18 @@ These become metavariables in the output.
 -/
 elab tk:"#check " colGt term:term : tactic => elabCheckTactic tk true term
 
+/--
+The `#check' t` tactic elaborates the term `t` and then pretty prints it with its type as `e : ty`.
+In contrast to `#check t`, we only pretty-print explicit arguments, and omit implicit or type class
+arguments.
+
+If `t` is an identifier, then it pretty prints a type declaration form
+for the global constant `t` instead.
+Use `#check' (t)` to pretty print it as an elaborated expression.
+
+Like the `#check'` command, the `#check'` tactic allows stuck typeclass instance problems.
+These become metavariables in the output.
+-/
+elab tk:"#check' " colGt term:term : tactic => elabCheckPrimeTactic tk true term
+
 end Mathlib.Tactic
```

Comment:
Making this a `Term` by `TSyntax.mk` fiat seems to *work*, but it's really surprising to me! Was this discussed with Kyle, or is it a pattern used elsewhere? I'm guessing it's patterned after the usage in `delabConstWithSignature`, but I don't really understand why it typechecks there either (where it doesn't need the constructor). If so, a comment explaining this and referring to `delabConstWithSignature` should be fine; I just don't want it to be mysterious for future readers. :)